### PR TITLE
add jupyter-structured scripts

### PR DESCRIPTION
**Pull request by lingjzhu**
This pull request adds the scripts to transform the jupyter notebooks from the stack into the `jupyter-structured` dataset used by Starcoder.

### CODE REVIEW

**Review by loubnabnl — APPROVED** [review]
Thanks, looks good! Can you add a comment about this to the readme?

File: preprocessing/jupyter-structured/jupyter-generate-triplets.py [hunk 1]
```diff
@@ -0,0 +1,529 @@
+<!DOCTYPE html>
```
Comment (loubnabnl) [inline preprocessing/jupyter-structured/jupyter-generate-triplets.py]: is this supposed to be a notebook?

Comment (lingjzhu) [reply preprocessing/jupyter-structured/jupyter-generate-triplets.py]: Sorry, it was an unintended mistake during copy/paste. I have updated it now.

**Comment by lingjzhu** [comment]
I have updated the readme.md files both in the main folder and the preprocessing folder. Correctly formatted scripts are now uploaded.

**Comment by loubnabnl** [comment]
Thanks!

**Pull request merged**